src/actions/channels.js

```diff
@@ -467,6 +467,35 @@ export function getChannelAndMyMember(channelId) {
     };
 }
 
+export function getChannelTimezones(channelId) {
+    return async (dispatch, getState) => {
+        dispatch({type: ChannelTypes.GET_CHANNELS_TIMEZONE_REQUEST}, getState);
+
+        let channelTimezones;
+        try {
+            const channelTimezonesRequest = Client4.getChannelTimezones(channelId);
+
+            channelTimezones = await channelTimezonesRequest;
+        } catch (error) {
+            forceLogoutIfNecessary(error, dispatch, getState);
+            dispatch(batchActions([
+                {type: ChannelTypes.GET_CHANNELS_TIMEZONE_FAILURE, error},
+                logError(error)(dispatch),
+            ]), getState);
+            return {error};
+        }
+
+        dispatch(batchActions([
+            {
+                type: ChannelTypes.GET_CHANNELS_TIMEZONE_SUCCESS,
+                data: channelTimezones,
+            },
+        ]), getState);
+
+        return {data: channelTimezones};
```

Comment:
I'm not sure how I feel about a Redux action that doesn't persist it's results to the store, but at the same time, I don't think it necessarily makes sense to add this data to the store since it's only used for this one-off action.

I think I'm good with how this is done for now since we can always move it into the store if we need to later